README.md

```diff
@@ -0,0 +1,27 @@
+# Combirepo
+
+Combirepo is a tool for combining several repositories into firmware.
+
+Intended to be used together with OBS-produced repositories.
+Refer man 1 combirepo for details.
```

Comment:
Maybe mark `man 1 combirepo` as a code snippet?

Reply:
yes, missed that